content/en/sections/common-requirements.md

```diff
@@ -1,9 +1,8 @@
-- Have strong experience using a version management system and continuous integration (CI) development process. We use `Git`/`Github` for version management and `Gitlab` for our `CI` pipelines. 
```

Comment:
Updates:

- got rid of version management from requirements and CI, I haven't seen a resume where someone doesn't mention some kind of CI + Github / Gitlab.
- Added "Drive success in a flat organization with minimal process, interfacing with technical and non-technical team members as necessary."